src/main/scala/io/radanalytics/streaming/amqp/AMQPFlowController.scala

```diff
@@ -287,12 +289,14 @@ private final class AMQPHrAsyncFlowController(
   private val scheduledExecutorService: ScheduledExecutorService = Executors.newScheduledThreadPool(1)
   private var scheduled: Option[ScheduledFuture[_]] = None
 
+  @transient private lazy val log = LoggerFactory.getLogger(getClass)
```

Comment:
is the enclosing class being serialized?  (in other words, why @transient?)

Reply:
I think it's not, I could have made it simple
```
private val log = LoggerFactory.getLogger(getClass)
```
but it shouldn't hurt, right?

If you want, I can change it.

Reply:
Yes, I agree with the move to non-spark logging to get it out of that namespace.  `@transient` is generally for fields in some class that is going to be serialized, but that particular field should be ignored during serialization (often combined with `lazy` so it gets properly recomputed on the other end of the serialization)